pynear/include/BuiltinSerializers.hpp

```diff
@@ -0,0 +1,128 @@
+#pragma once
+
+#include <Eigen/Core>
```

Comment:
why do we need this dependency? We shouldn't need eigen to compile

Reply:
Fixed on d5bc3b07fdb6716746872ca15eb496c2238bb240